Tests/IntegrationTests/TestDistribution/composer.json

```diff
@@ -6,7 +6,7 @@
         "bin-dir": "bin"
     },
     "require": {
-        "neos/neos-ui": "^5.0",
+        "neos/neos-ui": "^5.0.0",
```

Comment:
Now it limits it to *UI* 5.0.x - if that was intended, fine. So, it is planned to have releases for every branch in sync? I.e. 5.1.x, 5.2.x…

Reply:
yes, absolutely. See https://github.com/neos/neos-ui#versioning

Reply:
the plan is to release UI and Neos exactly in sync, except for the bugfixes